addons/OPT/GELDZEIT/fn_beam.sqf

```diff
@@ -106,18 +106,17 @@ switch GVAR(Fraktionauswahl) do
 
     default
     {
-        ERROR_LOG("GeldzeitBeam: Fehlerhafte Daten�bergabe - Keine Fraktionauswahl erkannt");
+        ERROR_LOG("GeldzeitBeam: Fehlerhafte Daten�bergabe - Keine Fraktionauswahl erkannt");
     };
 };
 
 if !(isNull _Basis) then
 {
     (["Basis"] call BIS_fnc_rscLayer) cutText ["Teleport...", "BLACK OUT", 3]; // fade out in black
-
     // beam player
-    vehicle player setPosASL [(random 100) - 50, (random 100) - 50, 1000 + random 100];
+    private _TempLogic = "Land_HelipadEmpty_F" createvehicle [(getPos _Basis select 0)-20*sin(random 360),(getPos _Basis select 1)-20*cos(random 360)];
     vehicle player setVectorUp [0,0,1];
```

Comment:
das ist gefährlich wenn das fahrzeug auf dem boden steht.

Reply:
Idee:

1. KFZ in 10m Höhe platzieren
2. KFZ der Terrain-Ebene anpassen
3. KFZ absenken

vehicle player setPosASL (getPosASL _TempLogic vectorAdd [0, 0, 10]);
vehicle player setVectorUp surfaceNormal position _TempLogic;
vehicle player setPosASL (getPosASL _TempLogic vectorAdd [0, 0, 0.2]);

Reply:
teste ich mal, hat mit den System eigentlich keine Probleme gemacht